app/models/nic/base.rb

```diff
@@ -62,6 +62,8 @@ class Base < ApplicationRecord
 
     belongs_to_host :inverse_of => :interfaces, :class_name => "Host::Base"
 
+    audited associated_with: :host
```

Comment:
we should ignore password (bmc interfaces), ideally it should use the same trick as e.g. compute_resource/image//user, see
https://github.com/theforeman/foreman/blob/develop/app/models/user.rb#L15
and https://github.com/theforeman/foreman/commit/518a2593744a7cdeaf3088f7fcf595b7cf6bd6f0 for an example

in this case we could just ignore it and hope that https://github.com/collectiveidea/audited/pull/339 will be finished soon

Reply:
![pass_exposed](https://user-images.githubusercontent.com/109773/31898994-c8372682-b81a-11e7-9930-81755d677a9b.png)

Reply:
@ares: This is how this looks for me:

![image](https://user-images.githubusercontent.com/4107560/31993497-991e814c-b97d-11e7-97b7-71b868b6a577.png)

Reply:
@timogoebel @ares Have you tried the latest version? I also see the password encrypted (like @timogoebel screenshot, or this if going from "no pass" to "some pass").
![screenshot from 2017-11-13 10-56-32](https://user-images.githubusercontent.com/598891/32719637-5c169ac6-c861-11e7-958e-85a57e1e9fbc.png)

Reply:
I wonder how our steps differ, but with I'm still able to reproduce with the latest version, I believe we should fix it the same way as I linked in previous comment at least to be consistent with other passwords in other objects